projects/documentation/src/components/styles.css

```diff
@@ -280,15 +280,6 @@ sp-sidenav:not(:defined) {
 sp-sidenav-item:not([tabindex]) {
     display: list-item;
     list-style-type: none;
-    margin: var(
-            --spectrum-sidenav-item-gap,
-            var(--spectrum-global-dimension-size-50)
-        )
-        0;
-}
-
-docs-page > sp-sidenav {
-    content-visibility: auto;
```

Comment:
@Westbrook any concerns about removing `content-visibility: auto;` ? 

With updating the width to 100% this was cutting off the focus-ring, removing it solved it. But I do not have full context as to why it was being used previously. 

Of note the property is [experimental and not currently compatible with safari.](https://developer.mozilla.org/en-US/docs/Web/CSS/content-visibility)

Reply:
@pfulton just so you have this info as well! ^^